pdpserver/Makefile

```diff
@@ -0,0 +1,21 @@
+build:
+	go build
+
+.PHONY: clean
+clean:
+	go clean 
+
+.PHONY: install
+install:
+	go install ./... 
+
+.PHONY: lint
+lint:
+	go get -v github.com/golang/lint/golint
```

Comment:
I ran on the setup, it still runs go get 
$ make lint
Using GOPATH=/Users/pankaj/go
go get -v github.com/golang/lint/golint
if [ -n "$(golint ./...)" ]; then \
		exit 1; \
	fi; \
$